include/nbl/builtin/glsl/transform_tree/debug/debug_draw.frag

```diff
@@ -0,0 +1,9 @@
+#version 440 core
+
+layout(location = 0) in vec4 inColor;
+layout(location = 0) out vec4 outColor;
+
+void main()
+{
+	outColor = inColor;
+}
```

Comment:
dont we have a shader like this somewhere already in `builtin/material` or `builtin/{Not GLSL}`

Reply:
probably yes, use it?

Reply:
I will replace it after merging this stuff